s3fs/tests/test_s3fs.py

```diff
@@ -519,14 +534,19 @@ def test_read_keys_from_bucket(s3):
                 s3.cat('s3://' + '/'.join([test_bucket_name, k])))
 
 
+@pytest.mark.skip(reason="misbehaves in modern versions of moto?")
```

Comment:
Should be xfail - I would expect it to run, and hopefully if will xpass in the future